Comparison shopping engine/Database.cs

```diff
@@ -108,25 +105,22 @@ public static async Task<List<Product>> Get(string table, string searchQuery = "
                 {
                     conn.Open();
 
-                    var cmd = new NpgsqlCommand("SELECT * FROM " + table + " where upper(name) like upper('%" + searchQuery + "%')", conn);
+                    var cmd = new NpgsqlCommand(
+                        "SELECT * FROM " + table + " where upper(name) like upper('%" + searchQuery + "%')", conn);
                     var dr = await cmd.ExecuteReaderAsync();
 
                     while (dr.Read())
-                    {
-                        pl.Add(new Product(dr[0].ToString(), dr[4] + "€", dr[2].ToString(), dr[3].ToString(), dr[1].ToString(), table));
-                    }
+                        pl.Add(new Product(dr[0].ToString(), dr[4] + "€", dr[2].ToString(), dr[3].ToString(),
```

Comment:
asmeniskai senas variantas iskaitomesnis

Reply:
Tiesa, aš irgi įpratęs po while ir if, for... matyti { } tačiau gerai kad bent į vieną eilutę nesudėjo :D Reik bandyti tobulinti akis matyt visam kodui, tai manau viskas bus gerai, gi ir mokomės kad tobulėtume